src/index.js

```diff
@@ -195,7 +195,12 @@ export default function (babel) {
       if (!tailPath) continue;
 
       if (tailPath.isExpressionStatement()) {
-        tailPath.replaceWith(getNewNode(tailPath.node.expression, tailPath));
+        const expression = tailPath.node.expression;
+        if (t.isAwaitExpression(expression)) {
```

Comment:
right now yes, this is implicit returns. if the user explicitly does `return <- foo` or `return await foo` it is kind of pointless, but maybe it's similar to `==` becoming `===`. up to you.

should I move this to `getNewNode()`?